R/JMModel.R

```diff
@@ -0,0 +1,42 @@
+cmdstan_mod <- R6::R6Class("CmdStanModel")
+setOldClass("CmdStanModel")
+
+#' @exportClass jmModel
+jmModel <- setClass("jmModel",
+    slots = c(cmdstan_mod = "CmdStanModel")
+)
+
+
+
+#' @importFrom cmdstanr write_stan_file
+#' @importFrom cmdstanr cmdstan_model
+#' @rdname StanModule-class
+#' @export
+setMethod(
+    f = "initialize",
+    signature = "jmModel",
+    definition = function(.Object,
+                          ...,
+                          Long,
+                          Os,
+                          name = "stan_model.stan") {
+        mod <- jm_complete(Long = Long, Os = Os)
```

Comment:
I think this could read nicely with `joint(long = long, os = os)` - I would avoid the uppercase argument names

Reply:
fixed